pkg/util/subnet/subnet.go

```diff
@@ -75,7 +75,14 @@ func (m *manager) GetHighestFreeIP(ctx context.Context, subnetID string) (string
 		return "", err
 	}
 
-	_, subnetCIDR, err := net.ParseCIDR(*subnet.AddressPrefix)
+	// grab the first addresPrefix in the subnet
+	var subnetCIDR *net.IPNet
+	if subnet.AddressPrefix == nil {
+		_, subnetCIDR, err = net.ParseCIDR((*subnet.AddressPrefixes)[0])
```

Comment:
is this always one or the other? and addressprefixes always has at least one element?

Reply:
yes